hasjob/views/helper.py

```diff
@@ -380,6 +380,27 @@ def get_post_viewcounts(jobpost_id):
     return values
 
 
+def get_max_counts(postids):
+    view_counts = [get_post_viewcounts(post.id) for post in JobPost.query.filter(JobPost.id.in_(postids))]
```

Comment:
This will hit the database. You've already got the viewcounts loaded from Redis. Use that and avoid the database.